receiver/vcenterreceiver/metadata.yaml

```diff
@@ -568,6 +568,27 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.vm.cpu.idle.time:
+    enabled: false
+    description: Total time that the CPU spent in an idle state.
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: [object_name]
+  vcenter.vm.cpu.wait.time:
+    enabled: false
+    description: Total CPU time spent in wait state.The wait total includes time spent the CPU Idle, CPU Swap Wait, and CPU I/O Wait states.
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: [object_name]
+  vcenter.vm.cpu.ready.time:
+    enabled: false
+    description: Milliseconds of CPU time spent in ready state.
```

Comment:
Nit: I "think" we try to avoid mentioning specific units in the description. I could be wrong here but if no other metric descriptions do this, might be a good idea to change it.

Reply:
I think you're right, looks bit odd. I will change it to just " CPU time spent in ready state", we have `unit` keyword to describe it. Thanks!